website/src/docs/hotchocolate/v13/migrating/migrate-from-12-to-13.md

```diff
@@ -1,3 +1,135 @@
+
+To restore the v12 default behavior, pass the [DbContextKind.Synchronized](/docs/hotchocolate/v13/integrations/entity-framework#dbcontextkindsynchronized) to the `RegisterDbContext<T>` call.
+
+**Before**
+
+```csharp
+services.AddGraphQLServer()
+    .RegisterDbContext<DbContext>()
+```
+
+**After**
+
+```csharp
+services.AddGraphQLServer()
+    .RegisterDbContext<DbContext>(DbContextKind.Synchronized)
+```
+
+> Note: Only add this, if your application requires it. You're better off with the new default otherwise.
+
+## DataLoaderAttribute
+
+Previously you might have annotated [DataLoaders](/docs/hotchocolate/v13/fetching-data/dataloader) in your resolver method signature with the `[DataLoader]` attribute. This attribute has been removed in v13 and can be safely removed from your code.
+
+**Before**
+
+```csharp
+public async Task<User> GetUserByIdAsync(string id, [DataLoader] UserDataLoader loader)
+    => await loader.LoadAsync(id);
+```
+
+**After**
+
+```csharp
+public async Task<User> GetUserByIdAsync(string id, UserDataLoader loader)
+    => await loader.LoadAsync(id);
+```
+
+## @authorize on types
+
+If you previously annotated a type with `@authorize`, either directly in the schema or via `[Authorize]` or `descriptor.Authorize()`, the authorization rule was copied to each field of this type. This meant that the authorization rule would be evaluated for each selected field beneath the annotated type in a request. This is not efficient, so we switched to evaluating the authorization rule **once** on the field that returns the annotated type instead.
+
+Lets imagine you currently have the following GraphQL schema:
+
+```graphql
+type Query {
+  user: User
+}
+
+type User @authorize {
+  field1: String
+  field2: Int
+}
+```
+
+This is how the authorization rule would be evalurated previously and now:
+
+**Before**
+
+```graphql
+{
+  user {
+    # The authorization rule is evaluated here, since this field is beneath
+    # the `User` type, which is annotated with @authorize
+    field1
+    # The authorization rule is evaluated here, since this field is beneath
+    # the `User` type, which is annotated with @authorize
+    field2
+  }
+}
+```
+
+**After**
+
+```graphql
+{
+  # The authorization rule is now evaluated here, since the `user` field
+  # returns the `User` type, which is annotated with @authorize
+  user {
+    field1
+    field2
+  }
+}
+```
+
+This behavior shouldn't be a breaking change regarding functionality, but the authorization error will now be raised at the field returning the type instead of the field beneath the type, which might break your test assertions.
```

Comment:
@michaelstaib I haven't played around with the new `@authorize` behavior so I'm not sure if this is correct. Could you look it over please?